hw/formal/tools/jaspergold/parse-formal-report.py

```diff
@@ -50,7 +50,7 @@ def format_percentage(good, bad):
     return '{0:.2f} %'.format(round(pc, 2))
 
 
-def parse_fpv_message(str_buffer):
+def parse_formal_message(str_buffer):
```

Comment:
nit: This should probably say `parse_log()`, since its the log file that is being parsed?